### PR TITLE
tests: Fix facts for iparole tests.

### PR DESCRIPTION
**Pull request by rjeffman**
This patch fixes setting ipaserver_domain and ipaserver_realm facts
for iparole tests, fixing variable evaluation and allowing the tests to be
executed even if `gather_facts: no`.

### CODE REVIEW

**Comment by rjeffman** [comment]
Please, do not merge before PR #631 is merged.

**Comment by t-woerner** [comment]
Don't we need this also in the other tests where ansible_facts is used?

**Comment by rjeffman** [comment]
Only if `gather_facts: no`, and. currently none, but iparole, failed. This was tested with RP #631, as the tests for client context disable `gather_facts`.

**Review by t-woerner — APPROVED** [review]
LGTM

**Pull request merged**
